### PR TITLE
Opal 163 view for kalendar settings

### PR DESCRIPTION
**Pull request by pataricza1987**

### CODE REVIEW

**Review by norvarju — DISMISSED** [review]
Feature works properly.

**Review by norvarju — APPROVED** [review]

**Review by norvarju — DISMISSED** [review]

File: app/src/main/res/drawable/border.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <stroke android:color="@color/BorderColor" android:width="1dp"/>
+    <corners android:radius="5dp" />
+</shape>
```
Comment (bekobarna) [inline app/src/main/res/drawable/border.xml]: put an empty line at the end, beacause github needs it

File: app/src/main/res/layout/activity_information_and_delete.xml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        android:layout_marginTop="16dp"
+        android:scrollbars="vertical"
+        android:background="@drawable/border"
+        app:layout_constraintBottom_toTopOf="@+id/button"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/mergedCalendarsTitle" />
+
+</android.support.constraint.ConstraintLayout>
```
Comment (bekobarna) [inline app/src/main/res/layout/activity_information_and_delete.xml]: new line here as well

File: app/src/main/res/layout/calendar_list_item.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        android:layout_marginEnd="8dp"
+        android:layout_marginLeft="8dp"
+        android:layout_marginRight="8dp"
+        android:layout_marginStart="8dp"
+        android:layout_marginTop="16dp"
+        android:text="TextView"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+</android.support.constraint.ConstraintLayout>
```
Comment (bekobarna) [inline app/src/main/res/layout/calendar_list_item.xml]: and here as well

File: app/src/main/res/layout/kalendar_item.xml [hunk 4]
```diff
@@ -25,4 +25,4 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
-</android.support.constraint.ConstraintLayout>
+</android.support.constraint.ConstraintLayout>
```
Comment (bekobarna) [inline app/src/main/res/layout/kalendar_item.xml]: here as well

**Review by bekobarna — APPROVED** [review]

**Review by norvarju — APPROVED** [review]

**Pull request merged**